src/constants.js

```diff
@@ -30,3 +31,4 @@ export const DEFAULT = {
 };
 
 export const WITHOUT_STR = "without"
+export const FAMILY_WITHOUT_STR = "without"
```

Comment:
Does creating 'FAMILY_WITHOUT_STR' make sense? 'WITHOUT_STR' seems quite general.

Reply:
On one hand yes, but I wanted to separate them - if for some reason WITHOUT_STR has to be changed for other feature they would have to be separated anyway.

Reply:
I don't think WITHOUT_STR will change. If it does, it won't be WITHOUT_STR anymore.